src/web-ui/src/analytics/AnalyticsHandler.js

```diff
@@ -839,4 +856,108 @@ export const AnalyticsHandler = {
     googleAnalyticsEnabled() {
         return process.env.VUE_APP_GOOGLE_ANALYTICS_ID && process.env.VUE_APP_GOOGLE_ANALYTICS_ID != 'NONE';
     },
+
+    evidentlyEnabled() {
```

Comment:
This makes more sense to be a constant at the top of the file since it will never change